managed/src/main/java/com/yugabyte/yw/models/InstanceType.java

```diff
@@ -179,6 +191,13 @@ public static void deleteInstanceTypesForProvider(Provider provider) {
       entries = entries.stream()
         .filter(supportedInstanceTypes(AWS_INSTANCE_PREFIXES_SUPPORTED))
         .collect(Collectors.toList());
+      for (InstanceType instanceType : entries) {
+        if (instanceType.instanceTypeDetailsJson.isEmpty()) {
+          instanceType.instanceTypeDetails.setVolumeDetailsList(
+            config.load().getInt(YB_VOLUME_INFO_VOLUME_COUNT),
```

Comment:
Inject config. Then no need to `load`.

Reply:
done

Reply:
If this is the only config  value then pass a parameter in to this method `awsDefaultVolumeCount` 
Do config lookup in caller.
I will do a refactor later which should fix this DI fail as described above - using singleton helper classes.

Reply:
Done!